boards/panasonic/panb511evb/panb511evb_nrf54l15-pinctrl.dtsi

```diff
@@ -75,4 +75,19 @@
 			low-power-enable;
 		};
 	};
+
+	/omit-if-no-ref/ grtc_default: grtc_default {
+		group1 {
+			psels = <NRF_PSEL(GRTC_CLKOUT_FAST, 1, 8)>,
+				<NRF_PSEL(GRTC_CLKOUT_32K, 0, 4)>;
```

Comment:
This conflicts with `uart30`: https://github.com/zephyrproject-rtos/zephyr/blob/5cb46afdbc06cd9e3ed0684b2a6082c6dc88469f/boards/panasonic/panb511evb/panb511evb_nrf54l15-pinctrl.dtsi#L34

Reply:
Hmm, Yeah you are right, what would you suggest? Is this a problem? The LFCLK will just be available on this pin if it was set in CLKOUT register (chapter 8.10.6.41 CLKOUT in datasheet) right? So in general it would not affect the UART, if the user did not enable the CLKOUT? Can we leave it this way, or do I have to remove the grtc_default for 32K completely?

Reply:
> Hmm, Yeah you are right, what would you suggest? Is this a problem? The LFCLK will just be available on this pin if it was set in CLKOUT register (chapter 8.10.6.41 CLKOUT in datasheet) right? So in general it would not affect the UART, if the user did not enable the CLKOUT?

Yes, the `clkout-32k` property needs to be set in the `grtc` node to activate this output. Without this, the UART will not be affected.

> Can we leave it this way, or do I have to remove the grtc_default for 32K completely?

It's up to you. I think I'd not assign this pin as clkout, but rather put a comment here that it is not done because by default the pin is used by the UART.

Reply:
Thanks for your suggestions! Did a rebase on your topics.